pkg/agent/datamodel/sig_config.go

```diff
@@ -268,6 +268,10 @@ func (d Distro) IsWindowsPIRDistro() bool {
 	return false
 }
 
+func (d Distro) IsWindowsDistro() bool {
+	return d.IsWindowsSIGDistro() || d.IsWindowsPIRDistro()
```

Comment:
is PIRDistro check still valid?

Reply:
in theory it could be for older clusters? figured I'd include it just to be on the safe side